### PR TITLE
YamiVersion.h & building checking & installation directory

### PR DESCRIPTION
**Pull request by clearlylin**

### CODE REVIEW

**Comment by clearlylin** [comment]
adjust included header file.

File: common/Makefile.am [hunk 1]
```diff
@@ -6,6 +6,10 @@ libyami_common_source_c = \
         nalreader.cpp \
         $(NULL)
 
+libyami_common_source_h = \
+        YamiVersion.h \
```
Comment (uartie) [inline common/Makefile.am]: Please use hard-tabs in makefiles

Comment (xuguangxin) [inline common/Makefile.am]: Hi @clearlylin : please update acroding uarite. so I can merge your first two patches.\
thanks

File: .gitignore [hunk 2]
```diff
@@ -17,3 +17,4 @@ autom4te.cache/
 m4/
 pkgconfig/
 aux/
+common/YamiVersion.h
```
Comment (chivakker) [inline .gitignore]: Do you care about the missing newline?

Comment (xuguangxin) [inline .gitignore]: yes, we can add a new line.

**Comment by uartie** [comment]
What about v4l2/Makefile.am?  It looks like v4l2_wrapper.h gets installed into $(includedir)/libyami_v4l2/.  Should v4l2_wrapper.h even get installed?  libyami_v4l2 does not have a .pc file.  Seems the libyami v4l2 install needs some attention.

File: common/Makefile.am [hunk 3]
```diff
@@ -24,13 +28,15 @@ libyami_common_cppflags = \
...
+noinst_HEADERS                = $(libyami_common_source_h_priv)
 libyami_common_la_SOURCES     = $(libyami_common_source_c)
 libyami_common_la_LDFLAGS     = $(libyami_common_ldflags)
 libyami_common_la_CPPFLAGS    = $(libyami_common_cppflags)
 
 DISTCLEANFILES = \
-	Makefile.in 
-
+	Makefile.in \
+	YamiVersion.h.in \
```
Comment (uartie) [inline common/Makefile.am]: This should be YamiVersion.h

We don't want to delete the YamiVersion.h.in file.

**Pull request closed without merging**